config/pipeline.yaml

```diff
@@ -0,0 +1,200 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# Configuration for running the complete pipeline including modeling and evaluation.
+
+# Absolute paths to non-code resources.
+file_paths:
+  queries: "/home/jupyter/github_msba_capstone/msba-capstone-2020/queries"
+
+# Global config used by multiple pipeline steps. Also made available
+# as parameters to all SQL queries.
+global:
+  # Project and dataset configuration.
+  source_project_id: "bigquery-public-data"
+  destination_project_id: "ut-goog"
```

Comment:
let's keep the master copy ready for delivery, which means taking our project ID out of the config file, since the user will have to provide it. remove ut-goog and add a comment about putting a value here.